doc/source/Contributing.rst

```diff
@@ -16,4 +16,32 @@ Adding a New Parameter
 
   2. Update the fortran definition for the ``grackle_chemistry_data`` type (in ``src/clib/grackle_fortran_interface.def``). This type must exactly match the definition of :c:data:`chemistry_data` and is used to integrate Grackle into simulation codes written in Fortran.
 
-  3. Add docummentation in ``doc/source/Parameters.rst`` for your new parameter.
+  3. Add documentation in ``doc/source/Parameters.rst`` for your new parameter.
+
+Instructions for Making a Release
+---------------------------------
+
+Before reading this section, please ensure you are familiar with our :ref:`versioning policy <versioning-code>`.
+
+To create a new release:
+
+  1. Draft the changes to the changelog and draft the release notes (GitHub allows you to circulate a draft of the release notes with other developers).
+
+  2. Create a new PR that includes 2 commits (commit A followed by commit B):
+
+     (i) Commit A (the first commit) is the final commit included in the release. It should:
+
+         - update the changelog
+	 - update the version number of the c-library. This is currently tracked in with `src/clib/Make.config.assemble` **AND** (for documentation purposes) in `doc/source/conf.py`
+         - update the version number of the python module (stored internally in `src/python/setup.py`)
+
+     (ii) Commit B (the second commit) is the first commit for the next version.
+          It should just update the version number of the c-library to specify the next development version
+
+	  -  like before, this must be updated in `src/clib/Make.config.assemble` **and** `doc/source/conf.py`
+          - do **NOT** touch anything else (even the python version number)
+
+  3. After the PR is merged, perform the release on GitHub.
+     Make sure to associate the release (and the new tag) with Commit A (not with Commit B).
+
```

Comment:
Another step is to go to the [readthedocs page](https://readthedocs.org/projects/grackle/) and add the release tag to list of versions to build.

Reply:
I missed that